src/Data/Formatter/Localizer/Drupal7.php

```diff
@@ -46,7 +46,9 @@ protected function localizeEntity($data)
                         // Localize Link fields - Hack.
                         // @TODO: Fix this for Links Type.
                         if (isset($fieldValue['uri'])) {
-                            $fieldValue['url'] = $fieldValue['uri'];
+                            // Checking if it is an internal URL. If it is,
+                            // then strip out the 'internal:/' prefix.
+                            $fieldValue['url'] = (substr($fieldValue['uri'], 0, 10) === 'internal:/') ? str_replace('internal:/', '', $fieldValue['uri']) : $fieldValue['uri'];
```

Comment:
@abarriosr I tested an internal link functionality locally. It works but it creates links at destination site using nids from source site. I think we should convert internal links into external replacing 'internal:/' part with source site uri.
```
'internal:/node/10' => 'http://source.site/node/10'
```